PlannTer/Modules/SettingsModule/SettingsView.swift

```diff
@@ -2,18 +2,27 @@ import SwiftUI
 
 struct SettingsView: View {
     @StateObject private var controller = SettingsController()
+    @State private var userName: String = ""
     
     var body: some View {
          ZStack {
-             Rectangle().fill(Color.primaryBackground)
-                 .edgesIgnoringSafeArea(.all)
-//            Color(.customPrimaryBackground)
-//                .edgesIgnoringSafeArea(.all)
+            Rectangle()
+                .fill(Color.primaryBackground)
+                .edgesIgnoringSafeArea(.all)
             VStack {
                 ScreenTitle(title: "Settings")
                     .padding(.top, 20)
                 Spacer()
-            } 
+                VStack(spacing: 20) {
+                    TextField("What should we call you?", text: $userName)
```

Comment:
There should be a text input here - the custom one we create. title and prompt should probably be preceeded by a label